.github/workflows/build_docs.yml

```diff
@@ -60,6 +65,12 @@ jobs:
     # The upload step should only be run on the main repository.
     if: github.repository == 'VowpalWabbit/vowpal_wabbit'
     steps:
+      - name: Set folder name to latest if push
+        if: github.event_name == 'push'
+        run: echo "::set-env name=FOLDER_NAME::latest"
+      - name: Set folder name to version if release
+        if: github.event_name == 'release'
+        run: echo "::set-env name=FOLDER_NAME::$(echo ${GITHUB_REF:10})"
```

Comment:
would be nice to get the vw version instead of the github hash, what would this imply in terms of extra work? might not be worth it

Reply:
maybe we could have the vw version in a version.h file (maybe we already do this) and get it using bash and be diligent on release process to manage that version

Reply:
some grep or awk

Reply:
This will extract the version from the tag, it wont be a hash. Seems counter intuitive I agree

Reply:
The reason this works is because it cuts out the first 10 chars of the ref "refs/tags/"

Reply:
This may be clearer though:
```yaml
- name: Get the version
  id: get_version
  run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/td-p/31595

Reply:
This is the same way we get the version in the docker-images repo

Reply:
got it, got fooled by the code